src/main/java/com/volmit/iris/core/ProjectManager.java

```diff
@@ -367,6 +363,10 @@ public void close() {
         }
     }
 
+    public File compilePackage(VolmitSender sender, IrisDimension d, boolean obfuscate, boolean minify) {
+        return new IrisProject(new File(getWorkspaceFolder(), d.getLoadKey())).compilePackage(sender, obfuscate, minify);
```

Comment:
Dont do this, it will add complexity to the back end. SImply use the parameter `dimension.getLoadKey()` using the old method, dont duplicate methods to change a single param in these classes

Reply:
One